xapian-core/include/xapian/query.h

```diff
@@ -257,6 +257,22 @@ class XAPIAN_VISIBILITY_DEFAULT Query {
 	 */
 	OP_EDIT_DISTANCE = 16,
 
+    /** Match only documents where a value slot is > a given value.
```

Comment:
This is still underindented — it's four spaces but the next one (added in this patch) is a tab.